src/connectedk8s/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'connectedk8s' Extension
+==========================================
+
+This package is for the 'connectedk8s' extension.
+i.e. 'az connectedk8s'
```

Comment:
Please add some introduction of the extension and examples to complete a full scenario. You can check the [README](https://github.com/Azure/azure-cli-extensions/tree/master/src/storage-preview) for storage-preview.